lib/beaker/hypervisor/abs.rb

```diff
@@ -74,5 +80,80 @@ def provision
     def cleanup
       # nothing to do
     end
+
+    def provision_vms(hosts)
+
+      verbose = false
+      config = Conf.read_config # get the vmfloaty config fiel in home dir
+
+      # TODO: the options object provided by the floaty cli is required in get_service_config()
+      # we should make it optional or accept nil
+      cli = Object.new
+      def cli.service() "abs" end
+      def cli.priority() "1" end # forces going ahead of queue
```

Comment:
Do we really want to default to priority 1 here?

Reply:
if ad-hoc users, yes. If CI, no. 
The original intent here is for ad-hoc users, but there is nothing stopping someone from hitting this code in CI. I don't know though how we would detect that.

Reply:
I think the way CI is configured right now is that there is always a resource_host defined. This method will only be called if that is nil, so I think we're reasonably safe from a correctly configured CI never getting this far into the logic.